tools/docker/Dockerfile

```diff
@@ -0,0 +1,121 @@
+FROM debian
```

Comment:
Set to debian:7.8 ; The debian 'latest' actually points to 7.4, but I just did a clean build and everything works.  Bumped the onl-build version to 1.5 to mark this change.  Thanks.